db/schema.rb

```diff
@@ -11,8 +11,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+
+ActiveRecord::Schema.define(version: 20160726170111) do
+
```

Comment:
Looks like we have 3 lines saying the same thing. I believe we only should have 1.